src/lib.js

```diff
@@ -80,34 +80,38 @@ function downloadVersion(version, os) {
     });
 }
 
-function cleanCurrentVersion () {
-	return glob('**/psc*', {cwd: paths.PSVM_CURRENT_BIN})
-	.then(function (files) {
-		return Promise.all(
-			R.map(function (file) {
-				return fs.unlink(path.join(paths.PSVM_CURRENT_BIN, file))
-			}, files)
-		);
-	});
+function cleanCurrentVersion() {
+    return glob('psc*', {
+            cwd: paths.PSVM_CURRENT_BIN
+        })
+        .then(function (files) {
+            return Promise.all(
+                R.map(function (file) {
+                    return fs.unlink(path.join(paths.PSVM_CURRENT_BIN, file))
+                }, files)
+            );
+        });
 }
 
 function use(version) {
     var srcPath = path.join(paths.PSVM_VERSIONS, version, 'purescript'),
         destPath = path.join(paths.PSVM_CURRENT_BIN);
 
-		cleanCurrentVersion()
-		.then(function () {
-			return glob('**/psc*', {cwd: srcPath});
-		})
-		.then(function (files) {
-			return Promise.all(
-				R.map(function (file) {
-					return util.copy(path.join(srcPath, file), path.join(destPath, file)).then(function () {
-						fs.chmodSync(path.join(destPath, file), '0777');
-					});
-				}, files)
-			);
-		});
+    cleanCurrentVersion()
+        .then(function () {
+            return glob('psc*', {
+                cwd: srcPath
+            });
+        })
+        .then(function (files) {
+            return Promise.all(
+                R.map(function (file) {
+                    return util.copy(path.join(srcPath, file), path.join(destPath, file)).then(function () {
+                        fs.chmodSync(path.join(destPath, file), '0777');
```

Comment:
Sorry, missed this the first time: since we're using promises everywhere else, perhaps it would make sense to return a promise here, i.e. using the asynchronous variant of `chmod` (which seems like it has already been promisified)?